tasks/deployMerkleDistributor.js

```diff
@@ -0,0 +1,23 @@
+require('@nomiclabs/hardhat-ethers')
+
+async function deployMerkleDistributor(tokenAddress, merkleRoot) {
+  const MerkleDistributor = await ethers.getContractFactory('MerkleDistributor')
+  const merkleDistributor = await MerkleDistributor.deploy(
+    tokenAddress,
+    merkleRoot
+  )
+  await merkleDistributor.deployed()
+  console.log(`merkleDistributor deployed at ${merkleDistributor.address}`)
+}
+
+task("deployMerkleDistributor", "Deploys the Merkle distributor contract")
+  .addPositionalParam("tokenAddress", "The airdropped token address")
+  .addPositionalParam("merkleRoot", "The merkle root hash")
+  .setAction(async (taskArgs) => {
```

Comment:
The `hre` argument is missing.
It might work if it's set as a global variable, however that might lead to unintended consequences.
```suggestion
  .setAction(async (taskArgs, hre) => {
```